### PR TITLE
minor fixes:

### PR DESCRIPTION
**Pull request by seanzhougoogle**
1. update mysql connector description
2. override postgres table query
3. handle oracle data type Number

### CODE REVIEW

File: oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleSourceDBRecord.java [hunk 1]
```diff
@@ -214,6 +214,11 @@ private void handleOracleSpecificType(ResultSet resultSet, StructuredRecord.Buil
         if (Double.class.getTypeName().equals(resultSet.getMetaData().getColumnClassName(columnIndex))) {
           recordBuilder.setDecimal(field.getName(), BigDecimal.valueOf(resultSet.getDouble(columnIndex)));
         } else {
+          // For a Number type without specified precision and scale, precision will be 0 and scale will be -127
+          if (precision == 0) {
+            // reference : https://docs.oracle.com/cd/B28359_01/server.111/b28318/datatype.htm#CNCPT1832
+            scale = 0;
```
Comment (yaojiefeng) [inline oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleSourceDBRecord.java:220]: why there is no precision = 38 here

File: oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnector.java [hunk 2]
```diff
@@ -129,4 +131,16 @@ protected String getTableQuery(DBConnectorPath path) {
...
     return String.format("SELECT * FROM %s.%s WHERE ROWNUM <= %d", path.getSchema(), path.getTable(), limit);
   }
+
+  @Override
+  protected Schema getSchema(int sqlType, String typeName, int scale, int precision, String columnName,
+                             boolean handleAsDecimal) throws SQLException {
+    // For a Number type without specified precision and scale, precision will be 0 and scale will be -127
+    if (precision == 0) {
+      // reference : https://docs.oracle.com/cd/B28359_01/server.111/b28318/datatype.htm#CNCPT1832
+      precision = 38;
```
Comment (yaojiefeng) [inline oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnector.java:141]: Do you know how the source handles this scenario, will the pipeline run fail or transform the record wrong?

File: oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleSourceDBRecord.java (hunk 1 above)
Comment (seanzhougoogle) [reply oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleSourceDBRecord.java:220]: precision is not used below

File: oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnector.java (hunk 2 above)
Comment (seanzhougoogle) [reply oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnector.java:141]: source will fail. I have the fix for source in this PR too.

Comment (seanzhougoogle) [reply oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleConnector.java:141]: OracleSourceSchemaReader and OracleSourceDBRecord are used by both connector and source

**Review by yaojiefeng — APPROVED** [review]

**Pull request merged**